### DISCUSSION TITLE
Add ability to manage groups

### DISCUSSION THREAD

**Discussion by Sanx78** [Jericho/ZoomNet#261] [Ideas]
Hi

Would be really good to be able to perform basic group operations within ZoomNet; enumerate groups on an account (so you can get group IDs), add and remove users from groups, etc. Management of the groups themselves is probably less important, but basic user/group functions would be really useful.

Thanks

**Comment by Jericho** [comment]
I agree and I created an [issue](https://github.com/Jericho/ZoomNet/issues/7) nearly four years ago to track progress on this new feature. Unfortunately, this feature has been put on hold due to the fact that I am not authorized to access the corresponding API endpoint. There are several other end points that I don't have access to, that's why I attempted to signup several times for Zoom's ISV partner program but I never received an answer. I even [asked the community for help](https://github.com/Jericho/ZoomNet/issues/160), but sadly nobody has been able to help.

At this point, I am relying on members of the community, like yourself, to work on such features and to submit PRs. If this feature is important to you, I encourage you to submit a PR and I'll be happy to include it in a future release.

**Comment by Sanx78** [comment]
Hi there

I will specifically raise this with our Zoom account manager and see if any traction can be gained that way. In the meantime, I’ll see if I’m anywhere a good enough programmer to add to your library via a PR.

Regards

Dan.
